modules/appeals_api/app/workers/appeals_api/higher_level_review_pdf_submit_job.rb

```diff
@@ -75,5 +74,27 @@ def receive_date(higher_level_review)
         .in_time_zone('Central Time (US & Canada)')
         .strftime('%Y-%m-%d %H:%M:%S')
     end
+
+    def log_upload_error(higher_level_review, e)
+      Rails.logger.info("#{higher_level_review.class.to_s.gsub('::', ' ')}: Submission failure",
+                        'source' => higher_level_review.consumer_name,
+                        'consumer_id' => higher_level_review.consumer_id,
+                        'consumer_username' => higher_level_review.consumer_name,
+                        'uuid' => higher_level_review.id,
+                        'code' => e.code,
+                        'detail' => e.detail)
+    end
+
+    def handle_upload_error(higher_level_review, e)
+      log_upload_error(higher_level_review, e)
+      higher_level_review.update(status: 'error', code: e.code, detail: e.detail)
+
+      if e.code == 'DOC201' || e.code == 'DOC202'
```

Comment:
Is it that any other error codes are related to the submitted content?

Reply:
Yeah. The 2– prefix is for issues with the CMP server. You can see a list of the codes here:
[lib/centrail_mail/upload_error.rn](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/central_mail/upload_error.rb)